Application/components/FriendsColumn/RemoveFriendModal.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { Modal, Text, Stack, Button, useMantineTheme, MultiSelect, Tooltip, Paper } from '@mantine/core';
+import { IconUserMinus } from '@tabler/icons-react';
+import { useUser } from '@clerk/nextjs';
+import { useCache } from '@/contexts/queryCacheContext';
+import { useFriendList } from '@/hooks/useFriendList';
+import { showNotification } from '@mantine/notifications';
+
+interface Friend {
+  id: string;
+  username: string;
+}
+
+interface RemoveFriendResponse {
+  message: string;
+}
+
+export function RemoveFriendModal(): JSX.Element {
```

Comment:
You don't actually need to specify `JSX.Element` here as the type (just a simple function will do). However, this could be good practice to indicate that what is being returned is a JSX element. This, however, is obviously the case.